aind-labtracks-service-server/src/aind_labtracks_service_server/handler.py

```diff
@@ -70,3 +75,47 @@ def get_subject_view(self, subject_id: Union[str, int]) -> List[Subject]:
         results = self.session.execute(statement=statement)
         subject_models = [Subject.model_validate(r) for r in results]
         return subject_models
+
+    def get_procedure_view(
+        self, subject_id: Union[str, int]
+    ) -> List[Procedure]:
+        """
+        Get a procedures view from LabTracks by joining several tables.
+        Parameters
+        ----------
+        subject_id : Union[str, int]
+          ID of mouse to pull information about.
+
+        Returns
+        -------
+        List[Procedure]
+          List of Procedure models. If more than one row is returned, then this
```

Comment:
It might be possible for a mouse to have multiple procedures

Reply:
Copy-paste mistake. I'll remove that comment